apps/data/src/tasks/data_upload_task.py

```
@@ -0,0 +1,450 @@
+        return
+
+    # Belt-and-suspenders SQL-literal quoting: the backend zod schema already
+    # restricts upload_table_name to [A-Za-z0-9_] and UUIDs are fixed-shape,
+    # but widgets can be set out-of-band (Databricks UI / manual run-now).
+    def quote(value: str | None) -> str:
+        if value is None:
+            return "NULL"
+        return "'" + value.replace("'", "''") + "'"
+
+    try:
+        completed_at = datetime.now()
```

Ruff (0.15.14)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)